src/ProjectTemplates/test/ProjectTemplates.Tests.csproj

```diff
@@ -56,7 +56,7 @@
     <ProjectReference Include="../Web.Client.ItemTemplates/Microsoft.DotNet.Web.Client.ItemTemplates.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="../Web.ItemTemplates/Microsoft.DotNet.Web.ItemTemplates.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="../Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj" ReferenceOutputAssembly="false" />
-    <ProjectReference Include="../Web.Spa.ProjectTemplates/Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="$(RepoRoot)src\submodules\spa-templates\src\Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj" ReferenceOutputAssembly="false" />
```

Comment:
This reference to Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj is insufficient to restore, build, or pack that template project when using `src\ProjectTemplates\build.cmd` or `src/ProjectTemplates/build.sh`. I had to modify those files ☹️